src/components/button/button.svelte

```diff
@@ -212,7 +213,7 @@
       color: var(--color-focus, var(--color));
       box-shadow: var(--box-shadow-focus);
       background: var(--bg-focus, var(--bg));
-      border-color: var(--border-color-focus, var(--border-color));
```

Comment:
should we remove all the `--border-color-focus` references (I don't think we use it now)

Reply:
Good call: https://github.com/brave/leo/pull/781/commits/788ae6c87c33a77368599ea02b4b6ca14d78b04a